status.py

```diff
@@ -2,21 +2,28 @@
 from cterasdk import *
 import csv, logging, os, re, sys
 
-def write_status(self,p_filename):
+def write_status(self,p_filename,all_tenants):
     """Save and write Filer status information to given filename."""
     get_list = ['config',
                 'status',
                 'proc/cloudsync',
                 'proc/time/',
                 'proc/storage/summary',
                 'proc/perfMonitor']
-    for filer in get_filers(self):
+    for filer in get_filers(self,all_tenants):
         info = filer.get_multi('', get_list)
+        tenant = filer.session().user.tenant
         sync_id = info.proc.cloudsync.serviceStatus.id
-        selfScanIntervalInHours = info.config.cloudsync.selfScanVerificationIntervalInHours
+        try:
+            selfScanIntervalInHours = info.config.cloudsync.selfScanVerificationIntervalInHours
+        except:
+            selfScanIntervalInHours = ('Not Applicable')
```

Comment:
Through this method, the parenthesis are not needed when you have only a string in them